assets/training/distillation/components/data_generation/src/generate_data.py

```diff
@@ -180,6 +181,14 @@ def get_parser():
         help="This enables Chain of Thought"
     )
 
+    parser.add_argument(
+        "--data_generation_task_type",
+        type=str,
+        required=True,
+        help="This helps in identifying the type of data generation task to generate the synthetic data.",
```

Comment:
Could we please add details of what all options are available & attached description to each of them?

Reply:
The ask is for customer to be able to see what all options are available. More importantly, have enough information about them to make an informed choice.